awx/ui_next/src/components/Lookup/Lookup.jsx

```diff
@@ -18,13 +27,27 @@ import SelectedList from '../SelectedList';
 import { ChipGroup, Chip } from '../Chip';
 import { getQSConfig, parseNamespacedQueryString } from '../../util/qs';
 
+const InputGroup = styled(PFInputGroup)`
+  ${props =>
+    props.multiple &&
+    `
+    --pf-c-form-control--Height: 90px;
+    overflow-y: auto;
+  `}
+`;
+
 class Lookup extends React.Component {
   constructor(props) {
     super(props);
 
+    this.assertCorrectValueType();
```

Comment:
We should have a test for this scenario

Reply:
I fought with this for a while. Jest has a `expect(X).toThrow()` but I can't get it to work as expected :/ No matter what I did, the error gets logged all over the test output

Reply:
For the component to work properly, it must have the correct value of `multiple`, which is determined by the type of the `value` property. Since we already enforce that `value` must either be an array or an array of `Item`s, I'm wondering if we actually need the `multiple` prop and the additional run-time validation here. 

Instead of passing it as a prop, what if we compute `multiple` based on whether or not `value` is an array? This would definitely add some additional / special behavior on the component we'd want to document, but it would remove the error path altogether.